### DISCUSSION TITLE
Disable multicast/forecast interface

### DISCUSSION THREAD

**Discussion by train-wreck** [strongswan/strongswan#1610] [Q&A]
I run Strongswan on an Arch Linux router with 2 network interfaces, lan and wan. I see the following entry in the Strongswan log:

> Mar 18 17:00:14 ROUTER charon-systemd[4614]: using forecast interface wan
Mar 18 17:00:14 ROUTER charon-systemd[4614]: joining forecast multicast groups: 224.0.0.1,224.0.0.22,224.0.0.251,224.0.0.252,239.255.255.250

I have no need for multicast on the WAN, how can I disable this? Thanks.

**Comment by tobiasbrunner** [comment]
Disable the `forecast` plugin (see [here](https://docs.strongswan.org/docs/5.9/plugins/pluginLoad.html)) or specify a particular interface in [the config](https://docs.strongswan.org/docs/5.9/plugins/forecast.html).
